PWA/package-lock.json

```diff
@@ -1008,6 +1008,11 @@
       "integrity": "sha512-shAmDyaQC4H92APFoIaVDHCx5bStIocgvbwQyxPRrbUY20V1EYTbSDchWbuwlMG3V17cprZhA6+78JfB+3DTPw==",
       "dev": true
     },
+    "@popperjs/core": {
```

Comment:
What is popperjs being used for? Is this module used on the code, or should it not be here?